apache/incubator-zeppelin
# Zeppelin-531 ]  Job management

### What is this PR for?
Zeppelin job management.
Zeppelin job monitoring and control page for Notebooks and Paragraphs.

You can check the cron notebook.
You can easily understand the entire current 'job.
It can easily run the Notebook necessary.
### What type of PR is it?
Feature

### Todo
- [x] notebook list and link.
- [x] notebook paragraph list and display status color and link.
- [x] fillter. - interpreter, name, 
- [x] user interpreter filter selecte box - dynamic list.
- [x] cron and normal notebook icon.
- [x] exception

- new
- [x] web app optimize for job manager
- [x] divede job manager socket.
- [x]  refresh job information on event base

### What is the Jira issue?
[ZEPPELIN-531](https://issues.apache.org/jira/browse/ZEPPELIN-531?filter=-1)

### How should this be tested?
click to job menu.
  - filter - write to notebook name and selected interpreter type.
  - check to error. -  Interpreter and problem notification about the execution impossible 'Notebook.
     For example, you run the R tutorial 'in the absence of R Interpreter'.
  - run always top option 
     The check box on the right exists.
     Try clicking while the notebook is running.
  - execute to notebook and stop
  - Click on the notebook name and paragraph to see if that move.

and using zeppelin. (make notebook, and run paragraph Misc...)

### Screenshots (if appropriate)
![jobm](https://cloud.githubusercontent.com/assets/10525473/15607235/437009b8-244c-11e6-908d-0922b473e15c.gif)
(Animated Gif, size 3.7MB)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes
